op-e2e/interop/testdata/emit.go

```diff
@@ -0,0 +1,368 @@
+// Code generated - DO NOT EDIT.
+// This file is a generated binding and any manual changes will be lost.
+
+package emit
```

Comment:
The package name is inconsistent with the testdata dir. And Go packages that are part of the build should not be in `testdata` (it's a special enshrined name that is ignored by the build tool in some places)

Reply:
Sure, changing it to "contracts"